addons/addon-base-raas/packages/base-raas-post-deployment/lib/steps/create-cloudfront-interceptor.js

```diff
@@ -31,115 +16,177 @@ class CreateCloudFrontInterceptor extends Service {
   }
 
   async execute() {
+    this.aws = await this.service('aws');
+    this.cloudFrontApi = new this.aws.sdk.CloudFront({ apiVersion: '2019-03-26' });
+    this.cloudFrontId = this.settings.get('cloudFrontId');
+    this.cloudFrontConfig = await this.cloudFrontApi.getDistributionConfig({ Id: this.cloudFrontId }).promise();
+    this.lambdaApi = new this.aws.sdk.Lambda({ apiVersion: '2015-03-31', region: 'us-east-1' });
```

Comment:
It's required to be `us-east-1` because Lambda@Edge only supports Lambda functions created in that region.

This could do with better documentation though. I've added a short comment to explain this.